cocos/animation/tracks/track.ts

```diff
@@ -379,6 +381,52 @@ export class TrackBinding {
         }
     }
 
+    public createRuntimeBindingAG (bindContext: AnimationClipGraphBindingContext) {
+        const {
+            origin,
+        } = bindContext;
+        const { path, proxy } = this;
+        const nPaths = path.length;
+        const iLastPath = nPaths - 1;
+
+        if (nPaths !== 0 && (path.isPropertyAt(iLastPath) || path.isElementAt(iLastPath)) && !proxy) {
+            const lastPropertyKey = path.isPropertyAt(iLastPath)
+                ? path.parsePropertyAt(iLastPath)
+                : path.parseElementAt(iLastPath);
+            const resultTarget = path[normalizedFollowTag](origin, 0, nPaths - 1) as any;
+            if (resultTarget === null) {
+                return null;
+            }
+
+            if (resultTarget instanceof Node && isTrsPropertyName(lastPropertyKey)) {
+                const transformPath = (() => {
+                    const segments = [] as string[];
+                    let node: Node | null = resultTarget;
+                    for (; node && node !== origin; node = node.parent) {
+                        segments.unshift(node.name);
+                    }
+                    if (node === origin) {
+                        return segments.join('/');
+                    } else {
+                        return undefined;
+                    }
+                })();
+                if (typeof transformPath === 'string') {
+                    const transformHandle = bindContext.bindTransform(transformPath);
+                    if (!transformHandle) {
+                        return undefined;
+                    }
+                    return bindPoseTransform(transformHandle, lastPropertyKey);
+                }
+            }
+        }
+
+        // !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
Q(@holycanvas ): Non-skeleton animations can not be used in animation graph?

A(@shrinktofit ): No, they can not be used as before.